pyQuARC/code/custom_validator.py

```diff
@@ -39,26 +39,14 @@ def ends_at_present_flag_presence_check(
     @staticmethod
     def mime_type_check(mime_type, url_type, controlled_list):
         result = {"valid": True, "value": mime_type}
-
-        if "USE SERVICE API" in url_type:
-            if mime_type:
-                result = StringValidator.controlled_keywords_check(
-                    mime_type, controlled_list
-                )
-
-        return result
-
-    @staticmethod
-    def data_center_name_presence_check(
-        archive_center, processing_center, organization_name
-    ):
-        result = {
-            "valid": False,
-        }
-        if value := archive_center or processing_center or organization_name:
-            result["valid"] = True
-            result["value"] = value
-
+        if url_type:
+            if "USE SERVICE API" in url_type:
```

Comment:
What if the `url_type` is something else? Sounds like a comment would help here

Reply:
added comments 7f3dd31